drivers/hmis/app/models/hmis/file.rb

```diff
@@ -35,11 +35,24 @@ def data_source
   scope :nonconfidential, -> { where(confidential: [false, nil]) }
 
   scope :viewable_by, ->(user) do
-    view_scope = where(client_id: Hmis::Hud::Client.with_access(user, :can_view_any_nonconfidential_client_files, :can_view_any_confidential_client_files))
-    # view_scope = view_scope.nonconfidential unless user.can_view_any_confidential_client_files?
-    edit_scope = user.can_manage_own_client_files? ? where(user_id: user.id) : none
+    client_scope = Hmis::Hud::Client
```

Comment:
same deal here I think we should add `viewable_by`. Even though it's probably not necessary in the context that we're currently using this scope, we should have it there to be safe. It will ensure that user has can_view_clients for those clients.